okta/help.md

```diff
@@ -568,6 +711,7 @@ by Okta themselves, or constructed by the plugin based on the information it has
 
 # Version History
 
+* 3.3.0 - New actions get_factors and send_push
```

Comment:
Use the title format per the style https://komand.github.io/python/style.html#changelog

```suggestion
* 3.3.0 - New actions Get Factors and Send Push
```